src/Events/ApplicationEvent.php

```diff
@@ -23,18 +24,20 @@ public function handle($event): void
         try {
             if ($breadcrumbEnabled || $eventEnabled) {
                 $payload = $this->getEventPayload($event);
+
+                // set requestId for all events, if available
+                $logContext = Log::sharedContext();
+                if (isset($logContext['requestId']) && !isset($payload->metadata['requestId'])) {
```

Comment:
I think having the request id _only_ in the events makes the most sense. We have talked about adding a request_id to the notice data, so we can correlate errors.

Reply:
Sorry, I should have been clearer in my request. We should remove `requestId` from the breadcrumbs, as it is not very useful, and only have it in the event payload.

I was thinking we would lean on requestId in the near term for cross-service correlation until we put effort into adding tracing.

Reply:
OK gotcha, fixed!